src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -693,11 +734,14 @@ def __write_complete_status_file(self):
             self.composite_logger.log_error("Core state file path returned a directory. Attempting to reset.")
             shutil.rmtree(self.complete_status_file_path)
 
-        # Write complete status file <seq.no>.complete.status
-        self.env_layer.file_system.write_with_retry_using_temp_file(self.complete_status_file_path, '[{0}]'.format(json.dumps(complete_status_payload)), mode='w+')
+        # Write complete status file <seq.no>.complete
```

Comment:
<seq.no>.complete.status

to avoid any confusions later

Reply:
done